hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -32,25 +32,71 @@
       default: "4",
       local:   "true"
     }
-    { name:    "NumLcStateBits",
+    { name:    "CsrLcStateWidth",
       desc:    "Number of life cycle state enum bits.",
       type:    "int"
       default: "4",
       local:   "true"
     }
-    { name:    "NumLcCntBits",
-      desc:    "Number of life cycle state enum bits.",
+    { name:    "CsrLcCountWidth",
+      desc:    "Number of life cycle transition counter bits.",
       type:    "int"
       default: "5",
       local:   "true"
     }
+    { name:    "CsrLcIdStateWidth",
+      desc:    "Number of life cycle id state enum bits.",
+      type:    "int"
+      default: "2",
+      local:   "true"
+    }
   ]
 
   /////////////////////////////
   // Intermodule Connections //
   /////////////////////////////
 
   inter_signal_list: [
+    // life cycle JTAG TAP
+    { struct:  "jtag"
+      type:    "req_rsp"
+      name:    "jtag"
+      act:     "rsp"
+      package: "rv_dm_pkg"
+    }
+    // Scanmode enable
+    { struct:  "logic"
+      type:    "uni"
+      name:    "scanmode"
+      act:     "rcv"
+      default: "1'b0"
+      package: ""
+    }
+    // Escalation inputs from alert handler
+    { struct:  "esc_tx"
+      type:    "uni"
+      name:    "esc1_tx"
```

Comment:
does it make sense to give these more unique names instead of 1/2? 
i guess in case in the future we expand the number of escalators for whatever reason.

Reply:
yeah that makes sense, let me rename those.